src/Nethermind/Nethermind.Facade/Eth/BlockForRpc.cs

```diff
@@ -73,10 +73,13 @@ public BlockForRpc(Block block, bool includeFullTransactionData, ISpecProvider s
         Size = _blockDecoder.GetLength(block, RlpBehaviors.None);
         StateRoot = block.StateRoot;
         Timestamp = block.Timestamp;
-        Transactions = (includeFullTransactionData
-                ? block.Transactions.Select((t, idx) => TransactionForRpc.FromTransaction(t, block.Hash, block.Number, idx, block.BaseFeePerGas, specProvider.ChainId))
-                : block.Transactions.Select(t => t.Hash).OfType<object>())
-            .ToArray();
+        TotalDifficulty = _isAuRaBlock ? block.TotalDifficulty ?? 0 : null;
```

Comment:
it is also for clique (like JOC):
```suggestion
        TotalDifficulty = block.Difficulty.IsZero ? null : block.TotalDifficulty ?? Uint256.Zero;
```